app/controllers/admin/error_traces_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Admin::ErrorTracesController < AdminController
+  before_action :require_power_user
+
+  class Boom < StandardError
+  end
+
+  def index
+    authorize { "ok" }
+  end
+
+  def create
+    authorize { "ok" }
+    if error_params.dig(:type) == "job"
+      raise_error = true
+      TracerJob.perform_async(Time.current.iso8601, raise_error)
+      redirect_to admin_error_traces_path, notice: "Background job error tracer submitted"
+    else
+      raise Boom, "Error test created by #{current_admin.full_name}"
+    end
+  end
+
+  private
+
+  def require_power_user
+    authorize { current_admin.power_user? }
```

Comment:
Ideally this would be 'technical staff' or something like that, but we don't have a user access level like that.  Power user is the best we can do.